src/configs/config.ts

```diff
@@ -58,6 +58,11 @@ export const generalRoninConf: GeneralConfig = {
     startedAtBlock: 11710199,
     bridgeContract: '0xCee681C9108c42C710c6A8A949307D5F13C9F3ca',
   },
+  [Network.Mainnet]: {
+    roninChainId: 2020,
+    startedAtBlock: 0, // TODO: update this variable
```

Comment:
@minh-bq

Reply:
Ronin code currently sets this to 23157000
https://explorer.roninchain.com/block/23157000